rtrlib/rtr/packets.c

```diff
@@ -1210,7 +1341,7 @@ static int rtr_sync_receive_and_store_pdus(struct rtr_socket *rtr_socket)
 					retval = RTR_ERROR;
 					goto cleanup;
 				}
-
+				
```

Comment:
remove

Reply:
clang-format is going to love all that whitespace